.travis.yml

```diff
@@ -13,5 +13,4 @@ script:
 
 after_success:
   - yarn run test-integration
-  - docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"
-  - ./scripts/ci-deployment/docker_ci.sh
+  - bash ./scripts/ci-deployment/deploy.sh
```

Comment:
That's because Windows checks in files by default with 644 permissions. Also some of the script didn't work with `sh`. I'll see if I can get them POSIX compliant.